cartographer/cloud/internal/testing/mock_local_trajectory_uploader.h

```diff
@@ -35,10 +35,10 @@ class MockLocalTrajectoryUploader : public LocalTrajectoryUploaderInterface {
   }
   MOCK_METHOD0(Start, void());
   MOCK_METHOD0(Shutdown, void());
-  MOCK_METHOD3(AddTrajectory,
-               void(int, const std::set<SensorId> &,
+  MOCK_METHOD4(AddTrajectory,
+               void(const std::string &, int, const std::set<SensorId> &,
```

Comment:
since you are touching this file you could change the style to "string& "

Reply:
done